src/main/java/gel/TaskList.java

```diff
@@ -0,0 +1,105 @@
+package gel;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import gel.exception.GelException;
+import gel.task.Deadline;
+import gel.task.Event;
+import gel.task.Task;
+import gel.task.Todo;
+
+/**
+ * Contains the task list e.g., it has operations to add/delete tasks in the list.
+ */
+public class TaskList {
+    private List<Task> listOfTasks;
+    private Ui ui;
+
+    public TaskList(Ui ui) {
```

Comment:
Should the public methods have header comments? I notice that public methods in some files do not have header comments.